.github/ISSUE_TEMPLATE/beta_feature.md

```diff
@@ -0,0 +1,59 @@
+---
+name: Beta Feature Requirements
+about: Requirements for promoting a feature to beta
+---
+
+This page lists the requirements for promoting a feature to beta. Please check off and document the steps as they are completed.
+
+**Feature Name:** 
+--- 
+
+### Requirements: 
+
+**Design**
+
+- [ ] Design doc has been approved
+
+**Config**
+
+- [ ] Can be enabled by default without requiring explicit user action. 
+
+**Docs** 
+
+- [ ] Performance expectations are documented, may have caveats. 
+- [ ] Documentation on istio.io includes samples/tutorials. 
+- [ ] Documentation on istio.io includes appropriate glossary entries. 
+- [ ] Where possible, all documentation changes have istio.io tests written. 
```

Comment:
Drop where possible here too, and we need to make this more explicit about what is different vs. alpha.

Reply:
Rephrasing this as "all new documentation containing user actions include istio.io tests". The phrasing here is weird, but for instance:
* the glossary can't be tested because it only contains text
* tutorials usually have lists of actions for a user to perform and can be tested
* reference docs can sometimes be tested